style/index.css

```diff
@@ -0,0 +1,63 @@
+/* index.css */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 60%;
+  font-family: Arial, sans-serif;
+  padding: 20px;
+  align-items: center;
+}
+
+h1 {
+  text-align: center;
+}
```

Comment:
- [ ] Please fix the below style issues to match the wireframe design.
  - Please make sure the entire book container has a border.
  - Please add the heading "Add a new book" top of the form.
  - Please make sure the form fields' width is less than the container of the book.
  - Please make sure submit button is aligned to the right instead of the center.

### Your design

![image](https://github.com/Obote/AwesomeBooks/assets/83803414/1d6fce97-0455-4407-8849-56c6393ecec1)

### Wireframe design

![image](https://github.com/Obote/AwesomeBooks/assets/83803414/6f67311f-39c4-46de-be36-16eb5381d55a)